base/strings/basic.jl

```diff
@@ -196,7 +196,6 @@ checkbounds(s::AbstractString, I::Union{Integer,AbstractArray}) =
 ## construction, conversion, promotion ##
 
 string() = ""
-string(s::AbstractString) = s
```

Comment:
This seems to change functionality: previously the function was the identity on any kind of string. After this does it convert general strings to `String`?